src/main/java/seedu/eylah/diettracker/model/food/Date.java

```diff
@@ -30,10 +30,26 @@ public Date(String date) {
         }
     }
 
+    public Date(LocalDateTime date) {
+        this.value = date;
+    }
+
     public String getStorageString() {
         return value.toString();
     }
 
+    public LocalDateTime getValue() {
+        return value;
+    }
+
+    public boolean isBefore(Date other) {
+        return value.isBefore(other.getValue());
+    }
+
+    public boolean isAfter(Date other) {
+        return value.isAfter(other.getValue());
+    }
+
```

Comment:
To be clearer, would be better to explain what exactly this isBefore/isAfter refers to.